compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -146,6 +146,11 @@ impl<'tcx> GotocCtx<'tcx> {
             let loc = self.codegen_span2(&ldata.source_info.span);
             let sym =
                 Symbol::variable(name, base_name, t, self.codegen_span2(&ldata.source_info.span));
+            if self.symbol_table.contains(&sym.name) {
+                debug!("Skipping already-added local: {:?}", sym.name);
```

Comment:
Why do we need this check?

Reply:
The spread arg is added in the prelude, so we shouldn't re-add it to the table here. But I added a different, more explicit check and comment instead, thanks!